tests/test_benchmarks.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from .utils import *
+
+TESTDIR = os.path.dirname(__file__)
+
+class BenchmarkTests(TranspileTestCase):
+
```

Comment:
At column 22: (F405) 'TranspileTestCase' may be undefined, or defined from star imports: .utils